default/generated/azure/32-openliberty-logging.yaml

```diff
@@ -0,0 +1,182 @@
+- category: mandatory
+  customVariables: []
+  description: Detect OpenLiberty Server Logging Configurations
+  effort: 3
+  labels:
+  - konveyor.io/source=java-ee
+  - konveyor.io/target=azure-aks
+  - domain=azure-readiness
+  - category=openliberty-migration
+  - openliberty
+  message: |
+    OpenLiberty server logging configurations detected that require optimization for Azure Monitor integration and containerized deployment in AKS.
+
+    **Azure Logging Strategy:**
+    • **Console Output:** Configure logs to stdout/stderr for Kubernetes collection by FluentD/Fluent Bit
+    • **Azure Monitor Integration:** Enable Application Insights for application performance monitoring
+    • **Log Analytics:** Use Azure Log Analytics workspace for centralized log aggregation and analysis
+    • **Structured Logging:** Implement JSON format for better parsing and querying in Azure Monitor
+
+    **Configuration Migration Steps:**
+    1. **Update server.xml:** Configure console logging with JSON format for Azure Monitor compatibility
+    2. **Environment Variables:** Use Kubernetes environment variables for dynamic logging configuration
+    3. **Azure Application Insights:** Integrate with Application Insights Java agent for automatic telemetry
+    4. **Log Forwarding:** Configure AKS log forwarding to Azure Monitor Logs
+    5. **Monitoring Setup:** Create Azure Monitor alerts and dashboards for application health
+
+  links:
+  - title: "Azure Monitor for containers"
+    url: "https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-overview"
+  - title: "Application Insights Java agent"
+    url: "https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-in-process-agent"
+  - title: "OpenLiberty logging configuration"
+    url: "https://openliberty.io/docs/latest/log-trace-configuration.html"
+  - title: "Kubernetes logging architecture"
+    url: "https://kubernetes.io/docs/concepts/cluster-administration/logging/"
+  ruleID: openliberty-logging-00001
+  when:
+    builtin.xml:
+      filepaths:
+      - server.xml
+      namespaces: {}
+      xpath: //server/logging
+
+- category: mandatory
+  customVariables: []
+  description: Detect OpenLiberty Log File Location Configurations
+  effort: 2
+  labels:
+  - konveyor.io/source=java-ee
+  - konveyor.io/target=azure-aks
```

Comment:
should we include other azure services in "konveyor.io/target"?

Reply:
Yes, let me add other services and update the message to make it not only for iaccess